Automattic/woocommerce-payments
# Added styling for instant deposit button and modal

Fixes #1141 #1139

#### Changes proposed in this Pull Request





Added styling of the instant deposit button and modal using existing wordpress/components, as mentioned in paJDYF-Rf-p2 . 
Due to the package is outdated, it seems only 2 of 3 button styles are available, primary and default (link), so these are the buttons that were used.

Added error/success notice to modal on error/success.
If notice is displayed, deposit button is removed.
If notice is displayed, closing the modal will refresh the page.

**Still needs to have instant deposit doc link added.**





#### Testing instructions





* This requires #1135 in order to test
* Go to the Express Dashboard and set your merchant account to use a Debit Card for payouts, instead of a bank account. Instant deposits only work with debit cards.
* Make some transactions in your store, so you have something to payout.
* Go to [$] Payments -> Deposits, you should see an Instant deposit link on the top right.
* That button will open a modal.
* I do not believe instant deposits are working correctly right now, so clicking the button for the deposit should result in an error, which should display the notice. 
* Closing the modal should then refresh the page.

-------------------
